src/style.css

```diff
@@ -0,0 +1,130 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
```

Comment:
Seems like you don't use classes from this file. Delete it.

Reply:
It looks like i don't have this file 
![image](https://user-images.githubusercontent.com/125855682/221960731-915412e1-c7a2-49fc-909f-235fe9a5a75c.png)